### ISSUE TITLE
kong proxy occasional high latency

### ISSUE THREAD

**Issue by soulseen** [Kong/kong#10329]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

kong 1.5.1

### Current Behavior

when I used this python script to test the app via the kong proxy:
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import uuid
import time
import json
import requests
import traceback

session = None


def check_session_available():
    global session
    if not session:
        print('Session would be created!!!!!!!!')
        session = requests.Session()
    else:
        return

check_session_available()

service_url = "http://xxx.xxx.xxx.xxx"
times = 1000

for i in range(times):
    json_params = {"inputs": []}

    for _ in range(2):
        tmp = {"inputs": [i * 1.0] * 3024}
        json_params["inputs"].append(tmp)
    traceid = str(uuid.uuid1())

    micro_model_start = time.time()
    headers = {"Content-Type": "application/json", "referer": traceid, 'Connection': 'keep-alive'}
    result = session.post(service_url + '/predict', json=json_params, headers=headers, timeout=0.5)
    resp_time = time.time() - micro_model_start
    print("code: ", result.status_code, "content: ", result.text)
    if resp_time > 0.5:
        print("traceid = {} timeout.".format(traceid))
        print("response time more than 0.5s")
        break
```
It will request the application 1000 times via the kong proxy, then randomly one response time will exceed 500ms and the rest will be less than 60ms.
I investigated the upstream services and the response times were all in the 50-70ms range, so my guess is that kong's packet return is slow

### Expected Behavior

kong response times were stable at under 60ms when tested

### Anything else?

I watch the proxy log but there is no error log.
here is the `tcpdump` result:
<img width="1434" alt="image" src="https://user-images.githubusercontent.com/16031013/220229509-8b6b1e78-9dd7-4dc6-845f-3453894de57e.png">
we can found the packet of number 4029 takes more than 1s.

**Comment by LL1024LL** [comment]
Did you request kong admin API ? We encountered similar problems before，and we fixed by related pr
- [https://github.com/Kong/kong/pull/8789](https://github.com/Kong/kong/pull/8789)

**Comment by sabertobihwy** [comment]
> kong 1.5.1

this version is too old ......

**Comment by soulseen** [comment]
> Did you request kong admin API ? We encountered similar problems before，and we fixed by related pr
> 
> * [fix(dao) fix incorrect function field reassignment #8789](https://github.com/Kong/kong/pull/8789)

I checked and have no request kong admin API.
after the deep Investigation, I found the TCP long connection between nginx and upstream was broken, and the new SYN packet was not acknowledged by upstream, so triggered the TCP Retransmission SYN packet resend and it delayed by approximately 1 second.
@sabertobihwy @LL1024LL

**Comment by hbagdi** [comment]
Good to close this issue in that case?
